mathlib_data_structures.py

```diff
@@ -0,0 +1,53 @@
+class mathlibStructures:
```

Comment:
Maybe, that would make sense to make those inherit from `enum.Enum` also?

Reply:
I did try using Enum at first, yes, but there was some issue - not sure what it was since I haven't focused on that in a while. Actually none of the classes here are used at the moment, this is going to be included in another PR which focuses on organizational structure of the python script itself, so that it's easier to read and update. I should probably remove this file from this PR and will look into using Enums again for the next one where those will be needed.